src/pages/Landing.js

```diff
@@ -29,8 +29,6 @@ export default function Landing() {
         shortening ailments that could hamper our quality of life.
       </p>
       <h3>Get started today on your new life today!</h3>
```

Comment:
If you are deleting these hyperlinks should we remove this header?

Reply:
I don't think the call to action necessarily needs to be removed (there isn't much content on the landing page, so anything helps), but I can go ahead and take that out too.

Reply:
The landing page content is pretty bare-bones for what is required there in the syllabus.